org.ekstep.contenteditorfunctions-1.2/editor/plugin.js

```diff
@@ -526,7 +526,7 @@ org.ekstep.contenteditor.basePlugin.extend({
         });
         var dialcodesUpdated = false;
         ecEditor._.forIn(nodesModified, function(node) {
-            if(node.metadata.dialcodes && !dialcodesUpdated){
+            if(node.metadata.dialcodes || node.metadata.dialcodes === null && !dialcodesUpdated){
```

Comment:
change this to 
if( (node.metadata.dialcodes || node.metadata.dialcodes === null )  && !dialcodesUpdated){
- && operator precedes || operator